core/src/main/scala/org/apache/spark/metrics/source/AccumulatorSource.scala

```diff
@@ -81,9 +82,10 @@ object LongAccumulatorSource {
  */
 @Experimental
 object DoubleAccumulatorSource {
-  def register(sc: SparkContext, accumulators: Map[String, DoubleAccumulator]): Unit = {
+  def register(sc: SparkContext, accumulators: Map[String, DoubleAccumulator]): Source = {
     val source = new DoubleAccumulatorSource
     source.register(accumulators)
     sc.env.metricsSystem.registerSource(source)
+    source
```

Comment:
What is the reason of changing this public API and exposing the package private `Source`?